src/benchmark/tpch_data_micro_benchmark.cpp

```diff
@@ -31,7 +33,7 @@ class TPCHDataMicroBenchmarkFixture : public MicroBenchmarkBasicFixture {
  public:
   void SetUp(::benchmark::State& state) override {
     auto& sm = Hyrise::get().storage_manager;
-    const auto scale_factor = 0.01f;
+    const auto scale_factor = 10.0f;
```

Comment:
Does the larger SF shadow anything we wanted to meaure with other benchmarks in this file?

Reply:
The smaller benchmark was just to allow running the micro benchmarks in the CI pipeline without too much waiting. However, we haven't executed those benchmarks automatically for years. :-/

Reply:
alright